src/mock/parsec/mod.rs

```diff
@@ -192,6 +192,10 @@ where
     }
 
     fn compute_consensus(&mut self, state: &mut SectionState<T, S::PublicId>) {
+        // Call `update_blocks` once, to allow this node to catch up to the previously consensused
+        // blocks.
+        let _ = self.update_blocks(state);
```

Comment:
Worth adding a test?

Reply:
There is already a failing test that is fixed by this change. But maybe better to have one focused specifically on this? I'll let you decide.

Reply:
Yes, that was a very difficult to reproduce random test, so probably best to have a unit test for it.

Reply:
Ok.